gluten-core/src/main/scala/org/apache/spark/sql/hive/HiveTableScanExecTransformer.scala

```diff
@@ -76,6 +76,8 @@ case class HiveTableScanExecTransformer(
     Seq.empty
   }
 
+  override def getRootPathsInternal: Seq[String] = Seq.empty
```

Comment:
Can we use table location as root path? Or add a TODO comment,

Reply:
Added TODO comment